test/utils/client.dart

```diff
@@ -26,6 +26,10 @@ String get testApiKey {
   return 'masterKey';
 }
 
+String? get openAiKey {
+  return Platform.environment['OPEN_AI_API_KEY'];
+}
+
```

Comment:
this will not work if the tests are run on web

Reply:
I suggest also adding documentation in CONTRIBUTING.md for users who want to run tests to set this env variable

Reply:
I've updated the code to cover flutter web, and updated the documentation in CONTRIBUTING.md file